src/pages/components/image-with-caption.mdx

```diff
@@ -29,6 +30,16 @@ The Image with caption supports images at 1x1, 2x1, 4x3, and 16x9 aspect ratios.
 </Column>
 </Row>
 
+## Content guidance
+
+| Field | Field type | Required   | Cardinality | Max character size (English / translated) |
+| -------- | --------- | ---------- | ---------- | ----------|
+| Headline (caption) | Short copy | Yes | 1 | XXS (40 / 55)|
+| Description | Long copy | No | 1 | L (150 / 200)|
```

Comment:
I would love it if we could find a way to be consistent in our naming across entities. (e.g. heading, and copy for instance)

Reply:
Absolutely agree! I'm thinking either "heading" or "headline" and either "paragraph" or "description". I think copy is too broad since a heading and headline are also copy. What do you think?

Reply:
Heading would be preferred since that's what I'm using everywhere in the documentation. As for paragraph or description let me think. Not everything is one of the two. It was broad on purpose.

Reply:
It is now "heading" :)